OpenEdXMobile/res/layout/activity_single_fragment_base.xml

```diff
@@ -6,65 +6,70 @@
     android:background="@color/grey_act_background"
     android:orientation="vertical"
     android:splitMotionEvents="false"
-    android:fitsSystemWindows="true"
-    >
+    android:fitsSystemWindows="true">
```

Comment:
I was curious about it, so gave it a try, turns out that it is better to have it due to the overall immersive experience.

Ref to the example below:

Without | With
--- | ---
![without_fitting](https://cloud.githubusercontent.com/assets/1710804/19079233/61c01936-8a6c-11e6-8c58-9907473c9310.png) | ![with_fitting](https://cloud.githubusercontent.com/assets/1710804/19079238/656f7eaa-8a6c-11e6-80f3-a3eef6a799e1.png)